### PR TITLE
Dev/platform config profile install

### PR DESCRIPTION
**Pull request by StephenHodgson**
https://github.com/XRTK/XRTK-Core/pull/564

### CODE REVIEW

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs [hunk 1]
```diff
@@ -17,9 +17,27 @@ internal static class OculusPackageInstaller
 
         static OculusPackageInstaller()
         {
-            if (!EditorPreferences.Get($"{nameof(OculusPackageInstaller)}", false))
+            EditorApplication.delayCall += CheckPackage;
+        }
+
+        [MenuItem("Mixed Reality Toolkit/Packages/Install Oculus Package Assets...", true)]
+        private static bool ImportLuminPackageAssetsValidation()
```
Comment (FejZa) [inline XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs:24]: Lumin? Probably copy paste issue here

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs [hunk 2]
```diff
@@ -17,9 +17,27 @@ internal static class OculusPackageInstaller
...
+        {
+            return !Directory.Exists($"{DefaultPath}\\Profiles");
+        }
+
+        [MenuItem("Mixed Reality Toolkit/Packages/Install Oculus Package Assets...")]
+        private static void ImportLuminPackageAssets()
```
Comment (FejZa) [inline XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs:30]: Lumin? Probably copy paste issue here

Comment (StephenHodgson) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs:30]: I don't see Lumin here...

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs (hunk 1 above)
Comment (StephenHodgson) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs:24]: Where do you see Lumin?

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs (hunk 2 above)
Comment (FejZa) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs:30]: It's saying `ImportLuminPackageAssets()`

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs (hunk 1 above)
Comment (FejZa) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs:24]: Same `ImportLuminPackageAssetsValidation()`

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs (hunk 2 above)
Comment (StephenHodgson) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs:30]: 👀 good catch lol

**Pull request merged**